plugin/impl.go

```diff
@@ -84,8 +84,22 @@ func (p *Plugin) Execute() error {
 			return err
 		}
 
+		isEnterprisePRCheck := false
+		if ossPR, ok := p.settings.params["OSS_PULL_REQUEST"]; ok && ossPR != "" && name == "grafana-enterprise" {
```

Comment:
I don't know if it's safe enough to do it that way or if I should add an explicit environment variable like `IS_ENTERPRISE_CHECK`

Reply:
Hm, I'd say better don't introduce another env variable - there will be so many flags for a single check, for something that we may end up deprecating _**eventually**_.